src/playback/tests/transport_tests.cpp

```
@@ -0,0 +1,1139 @@
+    EXPECT_CALL(*m_selectionController, leftMostSelectedItemStartTime())
+    .WillOnce(Return(std::nullopt));
+    EXPECT_CALL(*m_selectionController, rightMostSelectedItemEndTime())
+    .WillOnce(Return(std::nullopt));
+
+    //! [GIVEN] No time selection
+    EXPECT_CALL(*m_selectionController, timeSelectionIsEmpty())
+    .WillOnce(Return(true));
+
+    //! [GIVEN] No loop region active
+    EXPECT_CALL(*m_player, isLoopRegionActive())
+    .WillRepeatedly(Return(false));
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DECLARE_FLAGS is a macro then please configure it.

(unknownMacro)